src/main/java/imageData/Energy.java

```diff
@@ -0,0 +1,86 @@
+package imageData;
```

Comment:
Package names should describe what's in the package.
`imageData` is too generic.